### PR TITLE
SAK-48709: Academic Alert-Assignments: Only the assignment name is given for when an assignment is published and graded

### PR DESCRIPTION
**Pull request by zhuoY121**
JIRA: https://sakaiproject.atlassian.net/browse/SAK-48709

See the screenshot.

SAK-48709-local

### CODE REVIEW

File: webcomponents/bundle/src/main/bundle/sui-notifications.properties [hunk 1]
```diff
@@ -8,3 +8,10 @@ no_notifications=No notifications
 connection_request_received=You received a connection request from {0}
 connection_request_accepted={0} accepted your connection request
 message_received={0} sent you a message
+assignmentCreated = Created/updated a new assignment "{0}" in "{1}"
```
Comment (adrianfish) [inline webcomponents/bundle/src/main/bundle/sui-notifications.properties:11]: You need to use the same format for these key names as the rest of the file. So, you would use assignment_created here.

File: webcomponents/tool/src/main/frontend/js/sui-notifications/sui-notifications.js [hunk 2]
```diff
@@ -110,6 +116,24 @@ class SuiNotifications extends SakaiElement {
     }
   }
 
+  _decorateAssignmentNotification(noti) {
+    if (noti.event === "asn.new.assignment" || noti.event === "asn.revise.access") {
```
Comment (adrianfish) [inline webcomponents/tool/src/main/frontend/js/sui-notifications/sui-notifications.js:120]: I like to use a blank line to start the function body, unless it is a one line function.

```suggestion

    if (noti.event === "asn.new.assignment" || noti.event === "asn.revise.access") {
```

File: webcomponents/tool/src/main/frontend/js/sui-notifications/sui-notifications.js [hunk 3]
```diff
@@ -110,6 +116,24 @@ class SuiNotifications extends SakaiElement {
...
+      noti.title = this.i18n.assignmentCreated.replace('{0}', noti.title).replace('{1}', noti.siteTitle);
+    } else if (noti.event === "asn.grade.submission") {
+      noti.title = this.i18n.assignmentSubmissionGraded.replace('{0}', noti.title).replace('{1}', noti.siteTitle);
+    }
+  }
+
+  _decorateAnnouncementNotification(noti) {
+    if (noti.event === "annc.new") {
```
Comment (adrianfish) [inline webcomponents/tool/src/main/frontend/js/sui-notifications/sui-notifications.js:128]:
```suggestion

    if (noti.event === "annc.new") {
```

**Review by adrianfish — CHANGES_REQUESTED** [review]
Nice job. Just a few changes need to be made.

File: webcomponents/bundle/src/main/bundle/sui-notifications.properties (hunk 1 above)
Comment (zhuoY121) [reply webcomponents/bundle/src/main/bundle/sui-notifications.properties:11]: Sure! Thanks for your review, Adrian. I was not aware of it, Lol.

**Review by adrianfish — APPROVED** [review]
Good job, Zhuo. Thanks.

**Pull request merged**